src/compile/render-dom/index.ts

```diff
@@ -288,7 +289,7 @@ export default function dom(
 		return true;
 	});
 
-	const reactive_stores = component.vars.filter(variable => variable.name[0] === '$');
+	const reactive_stores = component.vars.filter(variable => variable.name[0] === '$' && variable.name !== '$$props');
```

Comment:
I'm a little nervous about having to worry about stuff like this whenever we want a list of store subscriptions, and also what will happen if we ever decide to add more `$$`-prefixed magic globals. Maybe a utility function we can use everywhere that just checks whether `str[0] === '$' && str[1] !== '$'`?

Reply:
Makes sense. Pushed another commit. A utility function is probably more trouble than it solves at this point, but if we find ourselves doing that a third time it's probably good to DRY it out when we get there